digital_land/phase/map.py

```diff
@@ -18,7 +23,9 @@ class MapPhase(Phase):
 
     def __init__(self, fieldnames, columns={}, log=None):
         self.columns = columns
-        self.normalised_fieldnames = {normalise(f): f for f in fieldnames}
+        self.normalised_fieldnames = {
+            normalise(f): replace_underscores(f) for f in fieldnames
```

Comment:
It does look weird, but it's building a dictionary where both the keys and values are fieldnames.
From what your suggesting above it would become 
normalise(f): normalise(f) for f...
I'll amend accordingly.

Reply:
Due to my comment above, I'll leave this the way it is for now.